ui/components/NodeInputSubmit.tsx

```diff
@@ -85,6 +88,21 @@ export const NodeInputSubmit: FC<NodeInputProps> = ({
           Use backup code instead
         </Button>
       )}
+      {showTotpLink && (
+        <Button
+          appearance="link"
+          tabIndex={5}
+          type="button"
+          onClick={() =>
+            (window.location.href = window.location.href.replace(
+              "&use_backup_code=true",
+              "",
```

Comment:
question: afaict this causes the page to reload, but we render the exact same flow. Can't we just re-render the page without making the calls to the backend? Would this make sense?

Reply:
yes, good idea. changed the approach a bit to avoid the refetch of the whole page.